glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/emccode/rexray
 import:
   - package: github.com/emccode/libstorage
-    ref:     v0.1.0-rc2
+    ref:     8d0aaaec8cc523697e6ee8de80a22585a4ab58e1
```

Comment:
Hi @clintonskitson,

I meant please omit the ref completely. It will default to the master branch if nothing is specified, and that's what we want for now.